test/test_helper.rb

```diff
@@ -1,3 +1,91 @@
 require "minitest"
 require 'simplecov'
 SimpleCov.start
+
+def lunch_data
+  [{ :district=>"Colorado",
+     :time_frame=>"2000",
+     :data_format=>"Percent",
+     :data=>"0.07",
+     :poverty_level=>"Eligible for Reduced Price Lunch" },
+   { :district=>"Colorado",
+     :time_frame=>"2000",
+     :data_format=>"Percent",
+     :data=>"0.27",
+     :poverty_level=>"Eligible for Free or Reduced Lunch" },
+   { :district=>"Colorado",
+     :time_frame=>"2000",
+     :data_format=>"Percent",
+     :data=>"0.2",
+     :poverty_level=>"Eligible for Free Lunch" }]
+end
+
+def economic_profile_file
+  {
+  :economic_profile => {
+    :median_household_income => "./data/Median household income.csv",
+    :children_in_poverty => "./data/School-aged children in poverty.csv",
+    :free_or_reduced_price_lunch => "./data/Students qualifying for free or reduced price lunch.csv",
+    :title_i => "./data/Title I students.csv"
+  }
+}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.